src/interaction/sessionrecording.cpp

```diff
@@ -409,16 +410,22 @@ bool SessionRecording::startPlayback(std::string& filename,
         LERROR("Unknown data type in header (should be Ascii or Binary)");
         cleanUpPlayback();
     }
-    // throwaway newline character
-    readHeaderElement(_playbackFile, 1);
+    // throwaway newline character(s)
+    std::string lineEnd = readHeaderElement(_playbackFile, 1);
+    bool isDosLineEnding = (lineEnd == "\r");
+    if (isDosLineEnding) {
+        // throwaway the second newline character (\n) also
+        readHeaderElement(_playbackFile, 1);
+    }
 
     if (_recordingDataMode == DataMode::Binary) {
         // Close & re-open the file, starting from the beginning, and do dummy read
         // past the header, version, and data type
         _playbackFile.close();
         _playbackFile.open(_playbackFilename, std::ifstream::in | std::ios::binary);
         size_t headerSize = FileHeaderTitle.length() + FileHeaderVersionLength
-            + sizeof(DataFormatBinaryTag) + sizeof('\n');
+            + sizeof(DataFormatBinaryTag) + sizeof('\n')
+            + ((isDosLineEnding) ? sizeof('\r') : 0);
```

Comment:
```suggestion
            + (hasDosLineEnding ? sizeof('\r') : 0);
```

Reply:
The line `+ (hasDosLineEnding ? sizeof('\r') : 0);` has now been removed (see next comment).